pilot/pkg/model/typed_xds_cache.go

```diff
@@ -195,6 +196,16 @@ func (l *lruCache[K]) updateConfigIndex(k K, dependentConfigs []ConfigHash) {
 	}
 }
 
+func (l *lruCache[K]) clearEvictQueue() {
+	// The underlying array releases references to elements so that they can be garbage collected.
+	var empty evictKeyConfigs[K]
```

Comment:
I meant to ask in my previous comment whether https://tip.golang.org/ref/spec#Clear will actually do the same with a single call.

Reply:
thanks~